src/include/sof/bit.h

```diff
@@ -24,5 +24,9 @@
 #define SET_BIT(b, x)		(((x) & 1) << (b))
 #define SET_BITS(b_hi, b_lo, x)	\
 	(((x) & ((1ULL << ((b_hi) - (b_lo) + 1ULL)) - 1ULL)) << (b_lo))
+#define GET_BIT(b, x) \
+	(((x) & (1UL << (b))) >> (b))
```

Comment:
I think this should be 1ULL.

Reply:
Correct, if you want to support more than 32 bits.

Reply:
It's now fixed.